packages/graphql/tests/integration/directives/cypher/filtering/relationships/cypher-filtering-relationship-auth-connection.int.test.ts

```diff
@@ -0,0 +1,380 @@
+/*
+ * Copyright (c) "Neo4j"
+ * Neo4j Sweden AB [http://neo4j.com]
+ *
+ * This file is part of Neo4j.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { TestHelper } from "../../../../../utils/tests-helper";
+
+describe("Connection API - cypher directive filtering - relationship auth filter", () => {
```

Comment:
I think you can remove the whole suite, and apply the suggestions for the other suite `describe("cypher directive filtering - relationship auth filter`, I believe what is worth testing is the `Connection` filtering style such as:`@authorization(filter: [{ where: { node: { actorsConnection_SOME: { name: "$jwt.custom_value" } }` that is not something supported! Another valid case to test using Connection Projection is for `field-leve auth` but that again is something not supported!